roles/network_bridge/tasks/network_manager.yml

```diff
@@ -0,0 +1,31 @@
+- block:
+    - name: create sysctl bridge config
+      template:
+        src: sysctl_bridge.conf
+        dest: /etc/sysctl.d/bridge.conf
+        owner: root
```

Comment:
Perhaps add a `group:` for explicitness.

Reply:
Done x3